io.github.najepaliya.kleaner.json

```diff
@@ -0,0 +1,115 @@
+{
+    "id": "io.github.najepaliya.kleaner",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-22.08",
+    "sdk": "org.kde.Sdk",
+    "command": "kleaner",
+    "finish-args": [
+        "--device=dri",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--filesystem=xdg-pictures"
```

Comment:
If you can use [QFileDialog](https://doc.qt.io/qt-5/qfiledialog.html) for the filechooser, it'd benefit from the portal and allow users add files from directories other than this.

Can be post merge on flathub, just a note.

Reply:
I have tried to keep all the UI elements in QML. This might be something I implement for the next release though. I'm probably going to need a  refactor for that anyway

Reply:
I don't exactly remember which is which but there's also another FileDialog type in `import Qt.labs.platform 1.1`, and also `QtQuick.Dialogs` were somewhat refactored in Qt 6 from what I heard.